src/pudl/etl.py

```diff
@@ -56,7 +56,8 @@ def _read_static_tables_eia() -> Dict[str, pd.DataFrame]:
         'prime_movers_eia': CODE_METADATA["prime_movers_eia"]["df"],
         'sector_consolidated_eia': CODE_METADATA["sector_consolidated_eia"]["df"],
         'fuel_transportation_modes_eia': CODE_METADATA["fuel_transportation_modes_eia"]["df"],
-        'contract_types_eia': CODE_METADATA["contract_types_eia"]["df"]
+        'contract_types_eia': CODE_METADATA["contract_types_eia"]["df"],
```

Comment:
Probably out of scope for this PR but does it make sense to  add a `source` key to `CODE_METADATA` so these static tables could be generated by just specifying a source? Something like this:

```python
def _read_static_table(source):
    return {table_name: metadata["df"] for table_name, metadata in CODE_METADATA.items() if metadata["source"] == source}
```

Reply:
This feels like it's similar to the categorization of the different Resource definitions in #1409

Reply:
Another related question is whether *all* static tables will be coding tables? Or if there are other types of static tables that we'll want to be able to include -- note that there are static non-coding tables in the FERC 1 dataset.